app/views/items/index.html.erb

```diff
@@ -6,13 +6,13 @@
 <% end %>
 <section id="most-popular-items">
 <p>Most Popular Items:</p>
-  <% Item.most_popular_items.each do |item| %>
+  <% @items.most_popular_items.each do |item| %>
```

Comment:
Thanks for changing this!  I'm not sure if I am actually right that class methods should only be in the model, but I appreciate the change